frontend/stores/list-watch.js

```diff
@@ -0,0 +1,41 @@
+import { observable, action } from 'mobx'
+import { createContext } from 'react'
+import axios from 'axios'
+
+class ListWatchStore {
```

Comment:
```suggestion
class WatchListStore {
```
Còn tên file này là `watch-list.js`